django/contrib/admin/options.py

```diff
@@ -109,6 +109,7 @@ class BaseModelAdmin(six.with_metaclass(forms.MediaDefiningClass)):
     formfield_overrides = {}
     readonly_fields = ()
     ordering = None
+    orderable_by = ()
```

Comment:
I'll suggest `sortable_by` as a possible alternative name to `orderable_by`. I think the use case Ramiro suggested would be good to support.